src/Application/GameLuaBindings.cpp

```diff
@@ -1,315 +1,108 @@
+                );
+            }
+            return luaState->create_table();
+        },
+        "set_character_info", [](int characterIndex, const sol::object &info) {
+            if(Character *character = pParty->getCharacterByIndex(characterIndex - 1); character != nullptr) {
+                const sol::table &table = info.as<sol::table>();
+                for (auto &&val : table) {
+                    std::string_view key = val.first.as<std::string_view>();
+                    if (key == "xp") {
+                        character->experience = val.second.as<int>();
+                    } else if(key == "sp") {
+                        character->uSkillPoints = val.second.as<int>();
+                    } else {
+                        logger->warning("Invalid key for set_character_info. Used key: {}", key);
+                    }
+                }
+            }
+        },
+        "play_all_characters_award_sound", []() {
+            for (auto &&character : pParty->pCharacters) {
+                character.PlayAwardSound_Anim();
+            }
+        },
+        "play_character_award_sound", [](int characterIndex) {
+            if(Character *character = pParty->getCharacterByIndex(characterIndex - 1); character != nullptr) {
+                character->PlayAwardSound_Anim();
             }
-            lua_pop(L, 1);
+        },
+        "go_to_screen", [](int screenIndex) {
+            SetCurrentMenuID(MenuType(screenIndex));
         }
-    } else {
-        return luaL_argerror(L, 2, lua_pushfstring(L, "Expected a table to set character info"));
-    }
-
-    return 0;
-}
-
-static int lua_play_character_award_sound(lua_State *L) {
-    lua_check_ret(lua_check_args_count(L, lua_gettop(L) == 1));
-    Character* character{};
-    lua_check_ret(lua_check_character_index(L, 1, character));
-    character->PlayAwardSound_Anim();
-    return 0;
-}
-
-static int lua_play_all_characters_award_sound(lua_State *L) {
-    for (auto&& character : pParty->pCharacters) {
-        character.PlayAwardSound_Anim();
-    }
-    return 0;
-}
-
-static int lua_get_party_size(lua_State *L) {
-    lua_pushinteger(L, pParty->pCharacters.size());
-    return 1;
-}
-
-static int lua_set_alignment(lua_State* L) {
-    lua_check_ret(lua_check_args_count(L, lua_gettop(L) == 1));
-    std::string_view alignmentStr = lua_tostring(L, 1);
-    PartyAlignment alignment;
-    if (tryDeserialize(alignmentStr, &alignment)) {
-        pParty->alignment = alignment;
-        SetUserInterface(pParty->alignment);
-    } else {
-        return luaL_argerror(L, 1, lua_pushfstring(L, "Invalid alignment value: '%s'", alignmentStr));
-    }
-    return 0;
-}
-
-static int lua_get_alignment(lua_State* L) {
-    lua_pushstring(L, toString(pParty->alignment).c_str());
-    return 1;
-}
+    );
+
+    /*
+    * Exposing serializations and deserializations functions to lua
+    * Useful for converting command line strings to the correct types
+    */
```

Comment:
Codestyle, comments inside functions usually start with `//` since it's terser this way.